### ISSUE TITLE
Design System Audit: Swap color with color variable in project.html

### ISSUE THREAD

**Issue by jdingeman** [hackforla/website#4138]
### Overview
During a reaudit of the Design Systems color variables, it was found that there was one color that was hardcoded in [project.html.](https://github.com/hackforla/website/blob/a9b9119133c884c251a0f5fe21704d8bad876025/_layouts/project.html#L171) We need to replace this with a color variable.

### Action Items
- [x] Refer to [/_sass/variables/_colors.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/variables/_colors.scss) for the current color variables (blues)
- [x] See [encycolorpedia.com for 336699](https://encycolorpedia.com/336699#:~:text=The%20hexadecimal%20color%20code%20%23336699,dark%20shade%20of%20cyan%2Dblue.) And name accordingly.  The name should not already exist, and define it as `#336699`
- [x] Replace the hardcoded color in project.html with the new variable you made in _colors.scss
  - [x] Check the website to ensure that this does not break the page it is used in
- [x] Once this PR is finished, please add the following note to the bottom of the issue in a comment
```
Design lead: please make a new issue to audit the design system and see if this color is part of our design system and if it is not, then identify if we should change it out, and what it should be. Or add to our design system.
```
### Merge Team/Tech Lead
- [ ] When this issue is closed, move this issue back to New Issue Approval and put a "Ready for design lead" label

### Resources/Instructions
Existing blue colors:
https://github.com/hackforla/website/blob/09d4b31f55f3ab8eea179e1805f1b1a292e56654/_sass/variables/_colors.scss#L31-L40

**Comment by jdingeman** [comment]
Marking as ready for prioritization

**Comment by jdingeman** [comment]
Marking as ready for prioritization

**Comment by perlaroyerc** [comment]
i. Availability: 11-6 ET M-F
ii. ETA: 3/11/23

**Comment by perlaroyerc** [comment]
Once this issue is closed.  This is a note for a Design lead: please make a new issue to audit the design system and see if this color is part of our design system. If it is not, identify if we should change it and what it should be or add to our design system.
